### ISSUE TITLE
The Low Ammo Alert volume does not change based on the "Alert volume" slider in the game settings.

### ISSUE THREAD

**Issue by LardumZ** [ReikaKalseki/Reika_FactorioMods_Issues#340]
#135 mentioned the same issue or atleast something similar. It seems the alerts in the mod don't react to the Alert volume slider in the game settings. 

And that sound gets incredibly fucking annoying really quickly, otherwise great mod.

**Comment by daniellovell** [comment]
Yes, this is an issue for me as well. Unmuteable alerts are incredibly annoying

Edit: It appears that the mixer bar that correlates to the Endgame Combat alerts are tied to "Game Effects" instead of "Alerts"

**Comment by gozulio** [comment]
Probably because they're being called as "game effects" instead of "alerts" or something. When the alert goes off while you're moving it can be heard fading off in the distance.
There's also the fact that the settings for these alerts either don't function intuitively (or at all) or don't apply to save games when you load them after having changed the setting.

**Comment by gozulio** [comment]
Right, So if you're willing to live without achievements (I don't think there's achievements in mudded games anyway) there is a way to gimp the turret monitoring alerts.

`/c game.player.force.technologies['turret-monitoring'].researched=false; `
Using this command in the console will un-research the Turret Monitoring tech, Thus making the alerts go away.

**Comment by KingAshoka846** [comment]
I tried the code, and it did remove the turret research, but the alerts did not go away and are still making noise. Is there another step I need to do? Thanks

**Comment by daniellovell** [comment]
Fix proposed here: https://github.com/ReikaKalseki/EndgameCombat/pull/9
